python/az/aro/azext_aro/custom.py

```diff
@@ -274,9 +274,14 @@ def get_route_tables_from_subnets(cli_ctx, subnets):
     for sn in subnets:
         sid = parse_resource_id(sn)
 
-        subnet = network_client.subnets.get(resource_group_name=sid['resource_group'],
-                                            virtual_network_name=sid['name'],
-                                            subnet_name=sid['resource_name'])
+        if not all(key in sid.keys() for key in ('resource_group', 'name', 'resource_name')):
+            logger.error("Failed to validate subnet '%s'", sn )
```

Comment:
This extra space should be removed 
```suggestion
            logger.error("Failed to validate subnet '%s'", sn)
```